LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2020
```

Comment:
License info was in the README.md badge, but I think adding this file is more suitable as a global convention (and know github sees this project as MIT-licensed).